kyo-core/shared/src/main/scala/kyo/loops.scala

```diff
@@ -17,18 +18,23 @@ object Loops:
     private val _continueUnit = Continue[Unit](())
 
     inline def continue[T]: Result[Unit, T] = _continueUnit
-    inline def done[T]: Result[T, Unit]     = ()
```

Comment:
The compiler doesn't line the `inline` overloaded methods anymore. This should be ok since the method will be inlined by the JIT compiler.